nixos-module.nix

```diff
@@ -19,12 +19,18 @@ let
   };
 
   containerOpts = types.submodule (import ./container.nix { inherit quadletUtils; });
-  networkOpts = types.submodule (import ./network.nix { inherit quadletUtils pkgs; });
+  networkOpts = types.submodule (import ./network.nix { inherit quadletUtils; inherit (cfg) podman; });
   podOpts = types.submodule (import ./pod.nix { inherit quadletUtils; });
 in
 {
   options = {
     virtualisation.quadlet = {
+      podman = mkOption {
```

Comment:
I think this option adds more confusion than usefulness.

IIUC, this controls the podman we use to 1. delete network; 2. generate rootless unit; while everything else still follows `config.virtualisation.podman.package` which sets `systemd.package` which links the generator into `/etc/systemd/system-generators/podman-system-generator`.

I'd say let's use `config.virtualisation.podman.package` unconditionally in nixos module.